src/Data/Repositories/ChannelOperationRequestRepository.cs

```diff
@@ -173,14 +176,7 @@ public async Task<(bool, string?)> AddUTXOs(IBitcoinRequest type, List<FMUTXO> u
                     .SingleOrDefaultAsync(x => x.Id == type.Id);
                 if (request != null)
                 {
-                    if (!request.Utxos.Any())
-                    {
-                        request.Utxos = utxos;
-                    }
-                    else
-                    {
-                        request.Utxos.AddRange(utxos.Except(request.Utxos));
-                    }
```

Comment:
Why has this been removed?

Reply:
This is unnecessary since we are taking the FMUTXOs from already saved ones. Previously, we wanted this, so we don't save many times the same UTXO, as far as I know.